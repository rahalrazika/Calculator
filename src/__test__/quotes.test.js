import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/quotes';

it('expect to render home components', () => {
  const result = renderer.create(<Quotes />).toJSON;
  expect(result).toMatchSnapshot();
});
