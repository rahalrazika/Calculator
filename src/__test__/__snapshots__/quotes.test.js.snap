// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render home components 1`] = `[Function]`;
